src/ethereum/cancun/fork.py

```diff
@@ -547,18 +554,25 @@ def apply_body(
         chain_id=chain_id,
         traces=[],
         excess_blob_gas=excess_blob_gas,
+        blob_versioned_hashes=(),
     )
 
-    process_message_call(system_tx_message, system_tx_env)
+    system_tx_output = process_message_call(system_tx_message, system_tx_env)
+
+    destroy_touched_empty_accounts(
+        system_tx_env.state, system_tx_output.touched_accounts
+    )
```

Comment:
I think we're going to try and rip out empty accounts entirely today, so hopefully we won't need this at all.